game.py

```diff
@@ -530,6 +686,36 @@ def _circle(self, r, cx, cy):
     def _footer(self):
         return '</svg>\n'
 
+    def save_highscore(self):
+        file_path = os.path.join(self._activity_root, 'data', 'highscore')
+        logging.debug(file_path)
+        highscore = []
+        highscore.append(0)
```

Comment:
can be compressed to 
```suggestion
        highscore = [0]
```